README.md

```diff
@@ -91,3 +91,8 @@ Questions & Comments
 --------------------
 
 For any and all feedback, please use the Issues on this repository. 
+
+License
+--------------------
+
+By downloading the .zip you are agreeing to the terms in the project [EULA](http://aka.ms/dotnet-cli-eula).
```

Comment:
Ok, I'll update the destination URL in the tool when we figure it out.

@blowdart I don't see how to make HTTPS in the URL shortener tool.